Xamarin.Essentials/Geolocation/Geolocation.ios.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CoreLocation;
+using Foundation;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Geolocation
+    {
+        internal static bool IsSupported
+            => CLLocationManager.LocationServicesEnabled;
+
+        static async Task<Location> PlatformLastKnownLocationAsync()
+        {
+            await Permissions.RequireAsync(PermissionType.LocationWhenInUse);
+
+            var manager = new CLLocationManager();
```

Comment:
Should we have this on the main thread or doesn't matter??

Reply:
It does not appear to matter.